sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusTestEnvironment.cs

```diff
@@ -156,23 +134,14 @@ public string BuildConnectionStringWithSharedAccessSignature(string entityName,
             return $"Endpoint={ParsedConnectionString.Endpoint};EntityPath={entityName};SharedAccessSignature={signature.Value}";
         }
 
-        /// <summary>
-        ///   Ensures that a Service Bus namespace is available.  If the <see cref="ServiceBusConnectionString"/> override was set for the environment,
-        ///   that namespace will be respected.  Otherwise, a new Service Bus namespace will be created on Azure for this test run.
-        /// </summary>
-        ///
         /// <returns>The active Service Bus namespace for this test run.</returns>
         ///
-        private NamespaceProperties ParseServiceBusNamespace(string serviceBusConnectionString)
+        private NamespaceProperties ParseServiceBusNamespace(string fullyQualifiedNamespace)
         {
-            var parsed = ServiceBusConnectionStringProperties.Parse(serviceBusConnectionString);
-
-            return new NamespaceProperties
-            (
-                parsed.Endpoint.Host.Substring(0, parsed.Endpoint.Host.IndexOf('.')),
-                serviceBusConnectionString.Replace($";EntityPath={parsed.EntityPath}", string.Empty),
-                false
-            );
+            int ending = ".servicebus.windows.net".Length;
```

Comment:
This may not work for sovereign clouds, as some use different domains.  (e.g. China, US Gov).    We should consider just finding the first dot (.) in the expression and slicing from there or doing a URL conversion and using the existing logic.

Reply:
Ok turns out this wasn't even used anymore so just deleted the whole thing